DatadogCore/Tests/DatadogObjc/DDSessionReplayOverridesTests.swift

```diff
@@ -0,0 +1,107 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+#if os(iOS)
+
+import XCTest
+import TestUtilities
+import DatadogInternal
+
+@testable import DatadogSessionReplay
+
+class DDSessionReplayOverrideTests: XCTestCase {
```

Comment:
@ganeshnj made a similar comment. I placed the extension in DatadogInternal because that's where the privacy levels are defined. Do you know what was the reasoning for placing them in the internal module initially?

Reply:
That's because the privacy levels are shared with the webview tracking module. But the override interface won't be shared.

Reply:
Got it, I'll move the extension to SR then.